src/libraries/Microsoft.Extensions.Logging.Console/src/ConsoleLoggerOptions.cs

```diff
@@ -11,34 +11,47 @@ namespace Microsoft.Extensions.Logging.Console
     /// </summary>
     public class ConsoleLoggerOptions
     {
-        private ConsoleLoggerFormat _format = ConsoleLoggerFormat.Default;
-
-        /// <summary>
-        /// Includes scopes when <see langword="true" />.
-        /// </summary>
-        public bool IncludeScopes { get; set; }
-
         /// <summary>
         /// Disables colors when <see langword="true" />.
         /// </summary>
+        [System.ObsoleteAttribute("ConsoleLoggerOptions.DisableColors has been deprecated. Please use ColoredConsoleLogFormatterOptions.DisableColors instead.", false)]
         public bool DisableColors { get; set; }
 
         /// <summary>
         /// Gets or sets log message format. Defaults to <see cref="ConsoleLoggerFormat.Default" />.
         /// </summary>
+        [System.ObsoleteAttribute("ConsoleLoggerOptions.Format has been deprecated. Please use ConsoleLoggerOptions.FormatterName instead.", false)]
         public ConsoleLoggerFormat Format
         {
-            get => _format;
+#pragma warning disable CS0618
+            get
+            {
+                if (FormatterName != null && FormatterName.Equals(ConsoleLogFormatterNames.Systemd, StringComparison.OrdinalIgnoreCase))
```

Comment:
>FormatterName [](start = 20, length = 13)

what happen if someone set the formatter name to json?

Reply:
the entire worflow is controlled with FormatterName. This code blob only makes sure the deprecated workflow still works using FormatterName